packages/app/src/pages/transaction/index.tsx

```diff
@@ -95,6 +100,14 @@ export const Transaction: React.FC = () => {
   };
 
   const setupWithState = async (tx: TransactionPayload) => {
+    if (tx.network) {
+      let network =
+        tx.network.version == TransactionVersion.Mainnet
+          ? new StacksMainnet()
+          : new StacksTestnet();
+      Object.assign(network, tx.network);
+      tx.network = network;
```

Comment:
Think this could be:

```suggestion
      tx.network = { ...network, ...tx.network };
```

Which avoids mutation of `network`, instead creating a new obj

Reply:
:+1:  addressed in https://github.com/blockstack/ux/pull/621/commits/0e0fea5f611b402fff934820e4775db5990b7979